src/app/Peerly/screens/AppreciationDetailsScreen/AppreciationDetailsComponent.tsx

```diff
@@ -134,8 +134,23 @@ const AppreciationDetailsComponent = ({
     }
   }, [isSuccessPostObjection, isObjectionModalVisible]);
 
+  useEffect(() => {
```

Comment:
Can we move these use effect calls to onSuccess and onError callbacks of useMutation ?